parsing/builtin_attributes.mli

```diff
@@ -115,8 +115,8 @@ val check_alerts_inclusion:
   def:Location.t -> use:Location.t -> Location.t -> Parsetree.attributes ->
   Parsetree.attributes -> string -> unit
 val alerts_of_attrs: Parsetree.attributes -> Misc.alerts
-val alerts_of_sig: Parsetree.signature -> Misc.alerts
-val alerts_of_str: Parsetree.structure -> Misc.alerts
+val alerts_of_sig: mark:bool -> Parsetree.signature -> Misc.alerts
+val alerts_of_str: mark:bool -> Parsetree.structure -> Misc.alerts
```

Comment:
The `mark` argument is only used as `mark:true`, is there a reason to keep this label?

Reply:
Not really, I added the label to indicate in the callsite that the function performs a side-effect (`alerts_of_str` sounds pure otherwise). But am happy to remove the argument if you think it is OK (or rename the function if you have any suggestions).

Reply:
That's a good point. Maybe `{str,sig}_cmi_alerts` to emphasize that we are extracting the alerts in order to store them in the cmi (and thus using them)?

Reply:
I think I prefer to keep the current version (with the labelled arguments) for now, which I find a bit more explicit even if not ideal either.

Reply:
Fine with me.